.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Expected behaviour:
```

Comment:
Can we add "brief description of problem" or something to that effect at the top?